test/fixtures/config/self_service_user.json

```diff
@@ -4,6 +4,7 @@
     "users": [
```

Comment:
Sounds like we're all agreed that this is not the ideal way to set-up this test data given the amount of duplication and difficulty to review for accuracy. Since you and Stephen are looking for a more efficient and easier to understand way to do this, lets move forward with this to unblock progress and make sure it is revisited later on. It seems using something like a builder with defaults and dynamically generating the `.json` (if required) might be good.